app/api/firebase/FirebaseAuthManager.ts

```
@@ -70,10 +70,10 @@
    * Sign in with Google
    */
   public async signInWithGoogle(): Promise<UserInfo> {
-    if (!this.app) throw new Error('Not connected to Firebase');
+    if (!this.getApp()) throw new Error('Not connected to Firebase');
 
     try {
-      const auth = getAuth(this.app);
+      const auth = getAuth(this.getApp()!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)